scripts/update_source_data_schema_changelog.py

```diff
@@ -0,0 +1,198 @@
+            "name": field_name_2,
+            "type": STRING
+        },
+    }
+    """
+    schema_json = read_json_file(os.path.join(directory, schema_name))
+    schema_json_by_col_name = {column["name"]: column for column in schema_json}
+    return schema_json_by_col_name
+
+
+def compare_schemas(schemas=[]) -> {}:
+    schema_changes = {}
+
+    for schema in schemas:
+        old_schema_json_by_col_name = get_mapped_schema_json(
+            directory=OLD_SCHEMAS_DIR, schema_name=schema
+        )
+        new_schema_json_by_col_name = get_mapped_schema_json(
+            directory=NEW_SCHEMAS_DIR, schema_name=schema
+        )
+
+        common, added, deleted = compare_lists(
+            old_list=old_schema_json_by_col_name.keys(),
+            new_list=new_schema_json_by_col_name.keys(),
+        )
+
+        type_changed = [
+            (
+                col,
+                old_schema_json_by_col_name[col]["type"],
+                new_schema_json_by_col_name[col]["type"],
+            )
+            for col in common
+            if old_schema_json_by_col_name[col]["type"]
+            != new_schema_json_by_col_name[col]["type"]
+        ]
+
+        if added or deleted or type_changed:
+            schema_changes[schema] = {
+                "column_added": added,
+                "column_removed": deleted,
+                "type_changed": type_changed,
+            }
+
+    return schema_changes
+
+
+def get_print_label_string(label: str) -> str:
+    return f"\n## {label}:\n" if label else label
+
+
+def get_print_schemas_string(label="", schemas=[]) -> str:
+    print_string = ""
+    if not len(schemas):
+        return print_string
+
+    print_string += get_print_label_string(label)
+    for schema in schemas:
+        print_string += f"- {schema.replace('_schema.json', '')}"
+    return print_string
+
+
+def get_print_schema_changes_string(label="", schema_changes={}) -> str:
+    print_string = ""
+    if not schema_changes:
+        return print_string
+
+    print_string += get_print_label_string(label)
+
+    markdown_table = "|       Table Name                | Operation     | Columns                  |\n"
+    markdown_table += "|---------------------------------|---------------|--------------------------|\n"
+
+    for schema_name, operations in schema_changes.items():
+        for operation, columns in operations.items():
+            if len(columns):
+                if operation in ["column_added", "column_removed"]:
+                    columns_str = ", ".join(columns)
+                if operation in ["type_changed"]:
+                    columns_str = ", ".join(
+                        [
+                            f"{column[0]} ({column[1]} -> {column[2]})"
+                            for column in columns
+                        ]
+                    )
+                markdown_table += f"| {get_print_schemas_string(schemas=[schema_name]):<33} | {operation:<15} | {columns_str:<50} |\n"
```

Comment:
```suggestion
                markdown_table += f"| {get_print_schemas_string(schemas=[schema_name]):<31} | {operation:<13} | {columns_str:<48} |\n"
```

nit: make the `markdown_table` column sizes consistent